tests/model/test_classified_failure.py

```diff
@@ -0,0 +1,33 @@
+from decimal import Decimal
+
+from treeherder.model.models import FailureMatch
+
+
+def test_set_bug(classified_failures):
+    rv = classified_failures[0].set_bug(1234)
+    assert rv == classified_failures[0]
+    assert classified_failures[0].bug_number == 1234
+
+
+def test_set_bug_duplicate(failure_lines, classified_failures, test_matcher):
```

Comment:
Could you add one or two comments inside this test to describe what you're testing?